native_client/ctcdecode/#ctc_beam_search_decoder.h#

```diff
@@ -0,0 +1,148 @@
+#ifndef CTC_BEAM_SEARCH_DECODER_H_
```

Comment:
looks like this file got in for no reason?

Reply:
@lissyx -- you mean the `if` got triggered when it shouldn't have?

Reply:
@JRMeyer no, I mean there is no reason this file should be here `#ctc_beam_search_decoder.h#` sounds like a temp artifact

Reply:
yup, this was an emacs temp file, deleted